daemon/mgr/spec_linux.go

```diff
@@ -65,6 +73,33 @@ func setupAppArmor(ctx context.Context, meta *ContainerMeta, spec *SpecWrapper)
 	return nil
 }
 
+func setupSeccomp(ctx context.Context, meta *ContainerMeta, spec *SpecWrapper) error {
+	if meta.HostConfig.Privileged {
+		return nil
+	}
+
+	// TODO: check whether seccomp is enable in your kernel, if not, cannot run a custom seccomp prifle.
+	seccompProfile := meta.SeccompProfile
+	switch seccompProfile {
+	case ProfileNameUnconfined:
+		return nil
+	case ProfilePouchDefault, "":
+		spec.s.Linux.Seccomp = seccomp.DefaultProfile(spec.s)
+	default:
+		spec.s.Linux.Seccomp = &specs.LinuxSeccomp{}
+		f, err := ioutil.ReadFile(seccompProfile)
```

Comment:
`ioutil.ReadFile` does not return a type of `os.File`, so I think we shall change the variable name from `f` into `data`, since its type is []byte. WDYT?

`f` would make readers to take it as an `os.File` typr for granted.